suricata.yaml.in

```diff
@@ -192,6 +193,7 @@ outputs:
         header: X-Forwarded-For
 
       types:
+        - krb5
```

Comment:
this will lead to error if no rust is enabled. Also, please move it down to be around the nfs/smb ones.